selene/factory.py

```diff
@@ -39,11 +40,11 @@ def is_driver_still_open(driver):
 
 def driver_has_started(name):
     shared_driver = get_shared_driver()
-    if not shared_driver:
+    if not shared_driver or isinstance(shared_driver, NoneObject):
```

Comment:
That's should still be just like `if not shared_driver` or `if not bool(shared_driver`

The problem was not in this if, but in NoneObject implementation - in python3  there is no `__nonzero__` but the `__bool__` should be used instead, and additional import is needed. This is the correct "compatible" version of NoneObject:

```
from builtins import object

class NoneObject(object):
    def __init__(self, description):
        # type: (str) -> None
        self.description = description

    def __getattr__(self, item):
        raise AttributeError("'NoneObject' for '%s' has no attribute '%s'" % (self.description, item))

    def __bool__(self):
        return False
```